### ISSUE TITLE
Option for measurement system isnt working

### ISSUE THREAD

**Issue by Kurokasa** [kandashi/Active-Auras#238]
Hi, thx for the awesome module.
The option in the module settings "Measurement system" doesnt work. It doesnt matter if the box ist checked or not, aura uses the systems setting. 

**To Reproduce**
1. Go to module settings and uncheck "Measurement system" so it should allways use euclidean as far as i understand the setting.
2. Set the system settings to (for DnD5e) PHB: Equidistant (5/5/5)
3. It still uses the system settings and doesnt override them. (i tried all combinations to see if i understood the setting wrong, the checkboxs does nothing)

**Expected behavior**
It should now use the euclidean measurement aka a circle, but it still is a square. 

![SharedScreenshot](https://user-images.githubusercontent.com/102412559/183912927-1d514703-55f4-47d0-ba8b-3c8fd7d9bde1.jpg)

**Environment:**
 - OS: Win10
 - Browser: chrome (forge more awesomeness)
 - Foundry Version 9
 - Module Version 0.4.13
 - All Modules Disabled: Yes

**Additional context**
I dont think it matters since i tried changing it, but the feature that has the effect as a passiv for the actor is set to sphere. It would be gr8 if that setting maybe changed stuff like that?

**Comment by ScootyMcBoogerBalls** [comment]
yeah, i have the same problem. I use the normal diagonal 5ft. movement rules. But also the Xanathars AoE Rules. Where atleast half a square has to be in the template to be considered.
And the default option here uses the movement measurement, which would be a square. But deactivating the option, still uses the same measurement.
